config/profiles.go

```diff
@@ -55,6 +56,10 @@ func Load(files ...string) (Profiles, error) {
 				continue
 			}
 
+			if strings.HasPrefix(name, "profile") {
```

Comment:
Would adding a trailing space be more consistent with the `TrimPrefix`?

```suggestion
			if strings.HasPrefix(name, "profile ") {
```